docs/home/howto/UPGRADE.md

```diff
@@ -268,9 +351,9 @@ Before upgrade procedure make sure you have a data backup!
 
 ---
 
-Since Epiphany v1.0.0 we provide upgrade elasticsearch-oss package to v7.10.2 and opendistro-\* plugins package to
+Since Epiphany v1.0.0 we provide upgrade elasticsearch-oss package to v7.10.2 and opensearch-\* plugins package to
 v1.13.\*. Upgrade will be performed automatically when the upgrade procedure detects your `logging`
-, `opendistro_for_elasticsearch` or `kibana` hosts.
+, `opensearch` or `kibana` hosts.
```

Comment:
What about opendistro_for_elasticsearch hosts?

Reply:
Can you pls clarify your point?